### PR TITLE
feat(grpc): add `Proposal` to `ConsensusInfo` API

### LINKED ISSUE #1437: Add Proposal to the ConsensusInfo API
## Describe the problem you'd like to have solved

We need to add proposal (or proposal data) to the [ConsensusInfo](https://github.com/pactus-project/pactus/blob/89938c4a1f63200fd86622148c745e7d6c34d9bb/www/grpc/proto/blockchain.proto#L289) API.

### PR DESCRIPTION
**Pull request by maxipaz**
## Description

Adding `Proposal` to the `ConsensusInfo` gRPC proto structure. 

## Related issue(s)

- Closes #1437

### CODE REVIEW

File: www/grpc/proto/blockchain.proto [hunk 1]
```diff
@@ -298,8 +298,24 @@ message ConsensusInfo {
...
+// Message containing information about a proposal.
+message Proposal {
+  // The height of the proposal.
+  uint32 height = 1;
+  // The round of the proposal.
+  int32 round = 2;
+  // The block data of the proposal.
+  string block_data = 3;
+  // The signature data of the proposal.
+  string signature_data = 4;
```
Comment (maxipaz) [inline www/grpc/proto/blockchain.proto:314]: Should we also include signature `PointG1`?

Comment (b00f) [reply www/grpc/proto/blockchain.proto:314]: No, Just Signature hex string is enough

File: www/grpc/proto/blockchain.proto [hunk 2]
```diff
@@ -298,8 +298,24 @@ message ConsensusInfo {
   int32 round = 4;
   // List of votes in the consensus instance.
   repeated VoteInfo votes = 5;
+  // The proposal of the consensus instance.
+  Proposal proposal = 6;
```
Comment (b00f) [inline www/grpc/proto/blockchain.proto:302]: All instances has the same proposal. So better we move it to `GetConsensusInfoResponse`

**Review by b00f — CHANGES_REQUESTED** [review]

File: www/grpc/proto/blockchain.proto [hunk 3]
```diff
@@ -188,6 +188,8 @@ message GetConsensusInfoRequest {}
 message GetConsensusInfoResponse {
   // List of consensus instances.
   repeated ConsensusInfo instances = 1;
+  // The proposal of the consensus info.
+  Proposal proposal = 6;
```
Comment (b00f) [inline www/grpc/proto/blockchain.proto:192]:
```suggestion
  Proposal proposal = 2;
```

**Review by b00f — COMMENTED** [review]
It's better to keep the numbers sequential.

File: www/http/blockchain.go [hunk 4]
```diff
@@ -250,5 +250,12 @@ func (s *Server) ConsensusHandler(w http.ResponseWriter, r *http.Request) {
 			tm.addRowBlockHash("BlockHash", v.BlockHash)
 		}
 	}
+
+	if res.Proposal != nil {
```
Comment (b00f) [inline www/http/blockchain.go:254]: Could you please move this before iterating on the consensus instances?

**Review by b00f — APPROVED** [review]

**Pull request merged**